README.md

```diff
@@ -1,8 +1,94 @@
-# Minetest app for YunoHost
+# Minetest for YunoHost
 
-- [Yunohost project](https://yunohost.org)
-- [Minetest website](http://www.minetest.net/)
+[![Integration level](https://dash.yunohost.org/integration/minetest.svg)](https://dash.yunohost.org/appci/app/minetest)
+
+[![Install Minetest with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=minetest)
+
+*[Lire ce readme en français.](./README_fr.md)*
+
+> *This package allow you to install Minetest quickly and simply on a YunoHost server.
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.*
+
+## Overview
+Minetest is a free open-source voxel game engine with easy modding and game creation.
+
+**Shipped version:** 5.1.0
+
+## Screenshots
+
+![](https://www.minetest.net/media/gallery/1.jpg)
+
+## Demo
+
+No demo but you can test with servers in https://servers.minetest.net/
+
+## Configuration
+
+How to configure this app: by SSH
+
+## Documentation
+
+ * Official documentation: https://wiki.minetest.net/
+ * YunoHost documentation: https://yunohost.org/#/minetest
+
+## YunoHost specific features
+
+#### Multi-users support
+
+Are LDAP and HTTP auth supported? No
+
+Can the app be used by multiple users? Yes
+
+#### Supported architectures
+
+* x86-64b - [![Build Status](https://ci-apps.yunohost.org/ci/logs/minetest%20%28Community%29.svg)](https://ci-apps.yunohost.org/ci/apps/minetest/)
+* ARMv8-A - [![Build Status](https://ci-apps-arm.yunohost.org/ci/logs/minetest%20%28Community%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/minetest/)
+* Jessie x86-64b - [![Build Status](https://ci-stretch.nohost.me/ci/logs/minetest%20%28Community%29.svg)](https://ci-stretch.nohost.me/ci/apps/minetest/)
+
+## Limitations
+
+* This app is not compatible with LDAP.
+
+## Additional information
+
+* Migration from old package
```

Comment:
What if people didn't see this message before clicking on upgrade? You could check the version in the upgrade script, and if the version is below 5.1.0, then abort the script, and print a message saying to look at the readme.

Reply:
Like this ?
```
if [ "$version = 0.01" ]; then
then
ynh_die --message="You can't upgrade to this version with simple upgrade. Please read the readme for upgrading https://github.com/YunoHost-Apps/minetest_ynh#additional-information..."
fi
```

Reply:
Copy past this code inside _common.sh:
https://github.com/YunoHost-Apps/Experimental_helpers/blob/0c1738e5078e98eb3f25220a0ae9f1de94b7b04b/ynh_read_manifest/ynh_read_manifest_2

and use `ynh_app_upstream_version` to fill version, so:

```
version=ynh_app_upstream_version

if [ "$version" = "0.01" ]; then
then
ynh_die --message="You can't upgrade to this version with simple upgrade. Please read the readme for upgrading https://github.com/YunoHost-Apps/minetest_ynh#additional-information..."
fi
```

should be ok